articles/data-analytics-using-pandas/index.md

```diff
@@ -0,0 +1,126 @@
+ Data science and Data Analytics are some of the hottest topics in the industry. The ability to analyze and make predictions based on data is nothing short of extraordinary. *Python is one of the most popular languages among the data science community. This is due to its ease of use and rich collection of libraries built to work with data. Pandas is one such library that makes handling data easy and efficient*. In this tutorial, we are going to understand how Pandas can be used to explore and draw insights from data. 
+
+### Table of Contents	 
+  
+- [Prerequisites](#prerequisites)
+- [What is Pandas?](#what-is-pandas)	
+- [Analysing data using Pandas](#analyzing-data-using-pandas)	 
+- [Code](#code)	
+- [Conclusion and further reading](#conclusion-and-further-reading)	 
+ 
+### Prerequisites
+
+1. A Basic understanding of programming in [Python](https://www.learnpython.org/)
```

Comment:
you could link an article from Engineering Education that touches on the basics of Python here